src/prefect/server/services/cancellation_cleanup.py

```diff
@@ -8,10 +8,15 @@
 from uuid import UUID
 
 import sqlalchemy as sa
+from docket import Depends as DocketDepends
```

Comment:
Since these modules aren't web services and importing FastAPI, is it necessary to alias `Depends`?